metrics_utility/automation_controller_billing/collectors.py

```diff
@@ -134,15 +132,14 @@ def config(since, **kwargs):
         'compliant': license_info.get('compliant'),
         'date_warning': license_info.get('date_warning'),
         'date_expired': license_info.get('date_expired'),
-        'subscription_usage_model': getattr(settings, 'SUBSCRIPTION_USAGE_MODEL', ''),  # 1.5+
+        'subscription_usage_model': settings_info.get('subscription_usage_model', ''),  # 1.5+
         'free_instances': license_info.get('free_instances', 0),
-        'total_licensed_instances': license_info.get('instance_count', 0),
-        'license_expiry': license_info.get('time_remaining', 0),
-        'pendo_tracking': settings.PENDO_TRACKING_STATE,
-        'authentication_backends': settings.AUTHENTICATION_BACKENDS,
-        'logging_aggregators': settings.LOG_AGGREGATOR_LOGGERS,
-        'external_logger_enabled': settings.LOG_AGGREGATOR_ENABLED,
-        'external_logger_type': getattr(settings, 'LOG_AGGREGATOR_TYPE', None),
+        'instance_count': license_info.get('instance_count', 0),
+        'pendo_tracking': license_info.get('pendo_tracking', ''),
+        'authentication_backends': license_info.get('authentication_backends', ''),
+        'logging_aggregators': license_info.get('logging_aggregators', ''),
+        'external_logger_enabled': license_info.get('external_logger_enabled', False),
+        'external_logger_type': license_info.get('external_logger_type', None),
```

Comment:
```suggestion
        'total_licensed_instances': license_info.get('instance_count', 0),
        'instance_expiry': license_info.get('time_remaining', 0),
        'pendo_tracking': settings_info.get('pendo_tracking_state'),
        'authentication_backends': settings_info.get('authentication_backends'),
        'logging_aggregators': settings_info.get('log_aggregator_loggers'),
        'external_logger_enabled': settings_info.get('log_aggregator_enabled'),
        'external_logger_type': settings_info.get('log_aggregator_type'),
```

(total_licensed_instances was renamed to instance_count, missing license_expiry, wrong setting name for pendo_tracking, and a bunch of settings were looking at license_info instead of settings_info or using the wrong name)